test/io-tests/fixtures.yaml

```diff
@@ -0,0 +1,23 @@
+roleclaims:
+  - key: ".postgrest.a_role"
+    data: {"postgrest": {"a_role": "postgrest_test_author"}}
+    expected_status: 200
+  - key: ".customObject.manyRoles[1]"
+    data: {"customObject": {"manyRoles": ["other", "postgrest_test_author"]}}
+    expected_status: 200
+  - key: '."https://www.example.com/roles"[0].value'
+    data: {"https://www.example.com/roles": [{"value": "postgrest_test_author"}]}
```

Comment:
what about using only single quotes in this yaml file? more consistent, because of line 8, where we need double quotes inside the string once.